driver/razermousemat_driver.h

```diff
@@ -13,12 +13,15 @@
 
 #define USB_DEVICE_ID_RAZER_FIREFLY_HYPERFLUX 0x0068
 #define USB_DEVICE_ID_RAZER_FIREFLY 0x0C00
+#define USB_DEVICE_ID_RAZER_FIREFLY_V2 0x0C04
```

Comment:
move the define below `USB_DEVICE_ID_RAZER_GOLIATHUS_CHROMA_EXTENDED`

Reply:
Done